frontend/src/components/pages/CampOverview/WaitlistedCampersTable.tsx

```diff
@@ -0,0 +1,206 @@
+import { SearchIcon } from "@chakra-ui/icons";
+import { FaEllipsisV } from "react-icons/fa";
+import {
+  Container,
+  HStack,
+  IconButton,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Table,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+  VStack,
+} from "@chakra-ui/react";
+import React from "react";
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faPerson,
+  faEnvelopesBulk,
+  faHourglassEnd,
+  faUserCheck,
+} from "@fortawesome/free-solid-svg-icons";
+import textStyles from "../../../theme/textStyles";
+
+const WaitlistStatusCard = ({ status, linkExpiry }: any) => {
```

Comment:
I think if there's a smaller component that's only being used in one component it's fine to define it in the same file - since these labels are going to be used in the registration list as well it would be a good idea to make it its own component in a file and export it - we would have to make the icon and text as props to pass in